bin/anthology/index.py

```diff
@@ -61,8 +61,9 @@ class AnthologyIndex:
     """Keeps an index of persons, their associated papers, paper bibliography
```

Comment:
This is a good time to extend the docstring for this constructor

Reply:
I agree, it would be helpful to have even roughly-written documentation for these classes; I often have to remind myself.